lightly/api/swagger_rest_client.py

```diff
@@ -1,9 +1,42 @@
+import json
 from typing import Any, Dict, Optional, Tuple, Union
 
 from lightly.openapi_generated.swagger_client.api_client import Configuration
+from lightly.openapi_generated.swagger_client.exceptions import (
+    ApiException,
+    UnauthorizedException,
+)
+from lightly.openapi_generated.swagger_client.models.api_error_code import ApiErrorCode
 from lightly.openapi_generated.swagger_client.rest import RESTClientObject
 
 
+class PrettyPrintApiException(ApiException):
+
+    def __init__(self, current_exception: ApiException):
+        super().__init__(current_exception.status, current_exception.reason)
+        self.body = current_exception.body
+        self.headers = current_exception.headers
+
+    def __str__(self):
+        error_message = "\n"
+        error_message += "#" * 30
```

Comment:
did you see this style somewhere or is this Malteshe schema? :) a bit extreme but at least its clear where an error starts and where it ends :)

Reply:
Guarin had proposed it: https://lightly-ai.slack.com/archives/C01KSD8EE65/p1712324976264859?thread_ts=1712242200.360349&cid=C01KSD8EE65

Reply:
GuarAlte code :)